dataflow_word_count.sh

```diff
@@ -0,0 +1,16 @@
+# chmod +x dataflow_word_count.sh
```

Comment:
Let's remove this file (I think the usage is in docstring)? Long-term we can add some dedicated docs on running dataflow.